ironfish-cli/src/commands/start.ts

```diff
@@ -204,6 +205,11 @@ export default class Start extends IronfishCommand {
       await this.firstRun(node)
     }
 
+    if (!node.internal.get('telemetryNodeId')) {
+      node.internal.set('telemetryNodeId', uuid())
+      await node.internal.save()
+    }
+
```

Comment:
We may have to move this to to the node later in case someone starts a node via the SDK.